build-tools/packages/build-cli/src/library/git.ts

```diff
@@ -465,7 +467,7 @@ export class Repository implements GitContext {
 	 * Fetch branch
 	 */
 	public async fetchBranch(remote: string, branchName: string): Promise<void> {
-		await this.gitClient.fetch(remote, [branchName]);
+		await this.gitClient.fetch(remote, branchName);
```

Comment:
Seems like we didn't catch this because there's a signature that can actually take a string array of options? Can we use the returned `FetchResult` to validate that what we want to happen actually happened? We know the branch name we want to fetch so I imagine that checking `FetchResult.branches` or `FetchResult.updated` could give us some confidence that this actually did what we want.

Reply:
I looked into this, and I don't think we can use the result for this. If the branch is already up-to-date with the
remote, the response looks like this:

```json
{
  'raw': '',
  'remote': 'https:\/\/github.com/microsoft/FluidFramework',
  'branches': [],
  'tags': [],
  'updated': [],
  'deleted': []
}
```

The fields are only populated when changes are fetched.